tests/unit_tests/cuda/surfaces_finder_cuda.cpp

```diff
@@ -0,0 +1,54 @@
+/** Detray library, part of the ACTS project (R&D line)
+ *
+ * (c) 2021 CERN for the benefit of the ACTS project
+ *
+ * Mozilla Public License Version 2.0
+ */
+
+#include <gtest/gtest.h>
+
+#include <climits>
+#include <cstdlib>
+#include <iostream>
+#include <vecmem/memory/cuda/managed_memory_resource.hpp>
+
+#include "surfaces_finder_cuda_kernel.hpp"
+
+TEST(surfaces_finder_cuda, surfaces_finder) {
+
+    // memory resource
+    vecmem::cuda::managed_memory_resource mng_mr;
+    vecmem::cuda::managed_memory_resource resource_array[n_grids];
```

Comment:
I'm pretty confused by this. I already didn't understand why you used an array of resources inside of the `detray::surfaces_funder` type.

But declaring an array of resources in the test is definitely wrong... You should just use the same single resource everywhere... You can "point to it" from multiple locations, fine, but it should really just be a single instance of the resource...

Reply:
It's just one of leftovers from my trials-and-errors. We definitely don't need this. Removed now